lib/evaluation/markForSubmissionEvaluationHandler.ts

```diff
@@ -18,20 +18,18 @@ export class MarkForSubmissionEvaluationHandler extends AbstractEvaluationHandle
   }
 
   public handle(pd: PresentationDefinition, p: VerifiablePresentation): void {
-    this.verifiablePresentation = new VP(
-      new Presentation(
-        null,
-        {
-          id: nanoid(),
-          definition_id: pd.id,
-          descriptor_map: [],
-        },
-        null,
-        [],
-        p.getHolder(),
-        null
-      )
-    );
+    this.verifiablePresentation = {
+      '@context': null,
+      type: null,
+      presentationSubmission: {
+        id: nanoid(),
+        definition_id: pd.id,
+        descriptor_map: [],
+      },
+      holder: p.holder,
+      verifiableCredential: null,
+      proof: null,
+    };
```

Comment:
we should rather create objects like
const ps = new presentationSubmission(params....)
new VerifiablePresentation (parms ... ps params ...)

Reply:
No he shouldn't